src/server/administer/index.md

```diff
@@ -8,7 +8,25 @@ Administration guide will help you to configure and maintain your <CommunityPlat
 
 See the list of all [environment variables](./environment.md) that you can configure.
 
-## Opt out of sending statistics
+## Telemetry Service
+
+<MainPlatformName /> telemetry service helps us gather some information about the usage of the platform, namely on-premise deployment. This information is very valuable to us to help us improve <MainPlatformName />.
+Please consider enabling this functionality, even if you are using <CommunityPlatformName /> edition. For <EnterprisePlatformName /> deployments, as you'll check down below, this is <b>mandatory</b>.
+
+The <MainPlatformName /> telemetry service (`call-home`) requires https access to `https://api.merginmaps.com/monitoring` on `TCP` port `443`.
```

Comment:
Does this service has stable IP address so when client want's to whitelists the api?

Reply:
Let's finish the discussion on Slack and create a new PR if needed